ui/apps/platform/src/reducers/featureFlags.js

```diff
@@ -1,18 +1,37 @@
 import { combineReducers } from 'redux';
 
-import { createFetchingActionTypes, createFetchingActions } from 'utils/fetchingReduxRoutines';
+import { createFetchingActionTypes } from 'utils/fetchingReduxRoutines';
+import { fetchFeatureFlags } from 'services/FeatureFlagsService';
+
+// Types
 
 export const types = {
     FETCH_FEATURE_FLAGS: createFetchingActionTypes('featureflags/FETCH_FEATURE_FLAGS'),
 };
 
-export const actions = {
-    fetchFeatureFlags: createFetchingActions(types.FETCH_FEATURE_FLAGS),
+// Actions
+
+export const fetchFeatureFlagsThunk = () => {
+    return async (dispatch) => {
+        dispatch({ type: types.FETCH_FEATURE_FLAGS.REQUEST });
+
+        try {
+            const result = await fetchFeatureFlags();
+            dispatch({
+                type: types.FETCH_FEATURE_FLAGS.SUCCESS,
+                response: result.response,
+            });
+        } catch (error) {
+            dispatch({ type: types.FETCH_FEATURE_FLAGS.FAILURE, payload: error });
```

Comment:
Good eyes. Especially because of comment below, here is a clear and simple naming convention:

```js
dispatch({ type: types.FETCH_FEATURE_FLAGS.FAILURE, error });
```

Reply:
We kept `response` from original code to limit initial scope to minimum changes.

A possible future improvement is to rename `response` property as either of the following:
* generic `data`  to undo confusing convention of service functions for sagas `(response) => ({ response: response.data })`
* specific `featureFlags` to eliminate extra level of property access, like in reducer below:
    `action.response.featureFlags` which could be `action.featureFlags` because we do not expect additional properties in the response of such a mature API endpoint